--- avidemux_2.6.8/cmake/admFFmpegBuild.cmake.orig	2015-04-13 12:49:22.200061063 +0100
+++ avidemux_2.6.8/cmake/admFFmpegBuild.cmake	2015-04-13 12:49:48.479401032 +0100
@@ -282,7 +282,6 @@
 						"${FFMPEG_BINARY_DIR}/libavutil/${LIBAVUTIL_LIB}"
 						"${FFMPEG_BINARY_DIR}/libpostproc/${LIBPOSTPROC_LIB}"
 						"${FFMPEG_BINARY_DIR}/libswscale/${LIBSWSCALE_LIB}"
-						"${FFMPEG_BINARY_DIR}/ffmpeg${CMAKE_EXECUTABLE_SUFFIX}"
 				   COMMAND ${BASH_EXECUTABLE} ffmpeg_make.sh WORKING_DIRECTORY "${FFMPEG_BINARY_DIR}")
 
 # Add and install libraries
